models/UserProfile.ts

```diff
@@ -2,21 +2,39 @@ import { Schema, model } from 'mongoose';
 
 const Role = {
   type: String,
-  enum: ['Patient', 'Parent', 'Clinician', 'Researcher'],
+  enum: ['researcher', 'patient', 'clinician', /*'Advocacy Group'*/],
```

Comment:
why comment?

Reply:
oh cuz Advocacy Group is in the spec and will be there eventually but there's no designs/icons for it yet

Reply:
i guess uncommenting it here is fine, as long as it's not in the portal code it wouldn't show up. and shoot forgot to change patient to patient/caregiver. coming up

Reply:
actually it's not letting the enums have spaces 💩